koku/masu/database/trino_sql/aws/openshift/managed_aws_openshift_daily.sql

```diff
@@ -0,0 +1,185 @@
+    bill_billingentity varchar,
+    lineitem_productcode varchar,
+    lineitem_availabilityzone varchar,
+    lineitem_lineitemtype varchar,
+    lineitem_usagetype varchar,
+    lineitem_operation varchar,
+    product_productfamily varchar,
+    product_instancetype varchar,
+    product_region varchar,
+    pricing_unit varchar,
+    resourcetags varchar,
+    costcategory varchar,
+    lineitem_usageamount double,
+    lineitem_normalizationfactor double,
+    lineitem_normalizedusageamount double,
+    lineitem_currencycode varchar,
+    lineitem_unblendedrate double,
+    lineitem_unblendedcost double,
+    lineitem_blendedrate double,
+    lineitem_blendedcost double,
+    pricing_publicondemandcost double,
+    pricing_publicondemandrate double,
+    savingsplan_savingsplaneffectivecost double,
+    bill_invoiceid varchar,
+    product_productname varchar,
+    product_vcpu varchar,
+    product_memory varchar,
+    resource_id_matched boolean,
+    matched_tag varchar,
+    source varchar,
+    ocp_source varchar,
+    year varchar,
+    month varchar,
+    day varchar
+) WITH(format = 'PARQUET', partitioned_by=ARRAY['source', 'ocp_source', 'year', 'month', 'day'])
+;
+
+-- Direct resource matching
+INSERT INTO hive.{{schema | sqlsafe}}.managed_aws_openshift_daily (
+    lineitem_resourceid,
+    lineitem_usagestartdate,
+    bill_payeraccountid,
+    lineitem_usageaccountid,
+    lineitem_legalentity,
+    lineitem_lineitemdescription,
+    bill_billingentity,
+    lineitem_productcode,
+    lineitem_availabilityzone,
+    lineitem_lineitemtype,
+    product_productfamily,
+    product_instancetype,
+    product_region,
+    pricing_unit,
+    resourcetags,
+    costcategory,
+    lineitem_usageamount,
+    lineitem_normalizationfactor,
+    lineitem_normalizedusageamount,
+    lineitem_currencycode,
+    lineitem_unblendedrate,
+    lineitem_unblendedcost,
+    lineitem_blendedrate,
+    lineitem_blendedcost,
+    pricing_publicondemandcost,
+    pricing_publicondemandrate,
+    savingsplan_savingsplaneffectivecost,
+    product_productname,
+    bill_invoiceid,
+    resource_id_matched,
+    matched_tag,
+    source,
+    ocp_source,
+    year,
+    month,
+    day
+)
+WITH cte_aws_resource_names AS (
+    SELECT DISTINCT lineitem_resourceid
+    FROM hive.{{schema | sqlsafe}}.aws_line_items_daily
+    WHERE source = {{aws_source_uuid}}
+        AND year = {{year}}
+        AND month = {{month}}
+        AND lineitem_usagestartdate >= {{start_date}}
+        AND lineitem_usagestartdate < date_add('day', 1, {{end_date}})
+),
+cte_array_agg_nodes AS (
+    SELECT DISTINCT resource_id
+    FROM hive.{{schema | sqlsafe}}.openshift_pod_usage_line_items_daily
+    WHERE source = {{ocp_source_uuid}}
+        AND year = {{year}}
+        AND month = {{month}}
+        AND interval_start >= {{start_date}}
+        AND interval_start < date_add('day', 1, {{end_date}})
+),
+cte_array_agg_volumes AS (
+    SELECT DISTINCT persistentvolume
```

Comment:
We will need to update this to include the `csi_volume_handle` that I just added for AWS 😓

Reply:
They should be one to one, so you should be safe to do a `select distinct persistentvolume, csi_volume_handle`